code/modules/mod/modules/module_kinesis.dm

```diff
@@ -15,27 +15,32 @@
 	overlay_state_inactive = "module_kinesis"
 	overlay_state_active = "module_kinesis_on"
 	accepted_anomalies = list(/obj/item/assembly/signaler/anomaly/grav)
+	/// Range of the knesis grab.
 	var/grab_range = 5
+	/// Time between us hitting objects with kinesis.
 	var/hit_cooldown_time = 1 SECONDS
-	var/movement_animation
+	/// Stat required for us to grab a mob.
+	var/stat_required = DEAD
+	/// How long we stun a mob for.
+	var/mob_stun_time = 5 SECONDS
+	/// Atom we grabbed with kinesis.
 	var/atom/movable/grabbed_atom
+	/// Ref of the beam following the grabbed atom.
 	var/datum/beam/kinesis_beam
+	/// Overlay we add to each grabbed atom.
 	var/mutable_appearance/kinesis_icon
+	/// Our mouse movement catcher.
 	var/atom/movable/screen/fullscreen/kinesis/kinesis_catcher
+	/// The sounds playing while we grabbed an object.
 	var/datum/looping_sound/gravgen/kinesis/soundloop
+	/// The cooldown between us hitting objects with kinesis.
 	COOLDOWN_DECLARE(hit_cooldown)
 
 /obj/item/mod/module/anomaly_locked/kinesis/Initialize(mapload)
 	. = ..()
 	soundloop = new(src)
 
 /obj/item/mod/module/anomaly_locked/kinesis/Destroy()
-	if(grabbed_atom)
```

Comment:
is none of this needed anymore?

Reply:
yeah, its handled by another piece of code